backend/src/main/java/moaon/backend/global/config/WebConfig.java

```diff
@@ -10,14 +10,15 @@ public class WebConfig implements WebMvcConfigurer {
     @Override
     public void addCorsMappings(CorsRegistry registry) {
         registry.addMapping("/projects/**")
-                .allowedOrigins(
-                        "http://localhost:3000",
-                        "https://moaon-dev.vercel.app",
-                        "https://d2ye9egv48bag1.cloudfront.net"
-                )
-                .allowedMethods("GET")
-                .allowedHeaders("*")
-                .allowCredentials(true)
-                .maxAge(3600);
+            .allowedOrigins(
+                "http://localhost:3000",
+                "https://moaon-dev.vercel.app",
+                "https://d2ye9egv48bag1.cloudfront.net",
+                "https://moaon-preview-*.vercel.app"
+            )
+            .allowedMethods("GET")
```

Comment:
```suggestion
            .allowedMethods("*")
```
method도 일단 다 열어두는거 어떨까요? 어차피 나중에 다 추가될 것 같아서 제안합니다